packages/prompts/src/index.ts

```diff
@@ -409,6 +414,27 @@ export const outro = (message = '') => {
 	process.stdout.write(`${color.gray(S_BAR)}\n${color.gray(S_BAR_END)}  ${message}\n\n`);
 };
 
+export type LogMessageOptions = {
+	symbol?: string;
+};
+export const log = {
+	message: (message = '', { symbol = color.gray(S_BAR) }: LogMessageOptions = {}) => {
+		process.stdout.write(`${color.gray(S_BAR)}\n${symbol}  ${message}\n`);
+	},
+	info: (message: string) => {
+		log.message(message, { symbol: color.blue(S_INFO) });
+	},
+	success: (message: string) => {
+		log.message(message, { symbol: color.green(S_SUCCESS) });
+	},
+	warn: (message: string) => {
```

Comment:
Although `console` calls it `warn`, I'd prefer `warning` (verb vs. noun). It reads better (as in you log a warning, not a "warn") and aligns better with the rest (which are all nouns).

Reply:
I see your point but I think this would trip me up since the equivalent `console.warn` is an API folks are already familiar with

Reply:
Fair enough. Would providing both/aliases be an option?

Reply:
Yeah go for it!